lib/Page.js

```diff
@@ -293,6 +296,21 @@ Page.prototype.insertSiteNav = function (pageData) {
     + '</div>';
 };
 
+Page.prototype.collectHeadFiles = function (pageData) {
+  const { head } = this.frontMatter;
+  if (!head) {
+    return pageData;
+  }
+  const headFilePath = path.join(this.rootPath, HEAD_FOLDER_PATH, head);
+  const headFileContent = fs.readFileSync(headFilePath, 'utf8');
+  // Map variables
+  const newBaseUrl = calculateNewBaseUrl(this.sourcePath, this.rootPath, this.baseUrlMap) || '';
+  const userDefinedVariables = this.userDefinedVariablesMap[path.join(this.rootPath, newBaseUrl)];
+  const headFileMappedData = nunjucks.renderString(headFileContent, userDefinedVariables);
```

Comment:
Is this rendering necessary? Seems to still work for me when I commented it out. Also the render method seems to be called again below.

Reply:
It is useful if the user predefined a `<style>` or `<link>` variable and used it in `headFile.md`.

The render method below is specifically for resolving `{{baseUrl}}`.